services/gcp/iam/roles.yml

```diff
@@ -17,7 +17,12 @@ privileges:
     notes: >-
       Deleting a custom role is possible even when it's present in bindings. The bindings remain, but are ineffectual.
   update:
-    risks: [escalation:privilege, impact:access]
+    risks: [escalation:lateral, escalation:privilege, impact:access]
```

Comment:
What's the lateral escalation here?

Reply:
You update a custom role and add permissions to another service. E.g. you add `storage.` permissions to a role you already have and contains only `compute.` permissions.

Reply:
I think here lateral movement would imply you gain access to a new role entirely, or perhaps new project.

Reply:
Being able to update your own role has the same effect as gaining access to a service account. Both can expand your permissions to new services. Updating a role is even more dangerous because you can pick your permissions.